Include/cpython/optimizer.h

```diff
@@ -12,7 +12,7 @@ typedef struct {
 } _PyVMData;
 
 typedef struct _PyExecutorObject {
-    PyObject_HEAD
+    PyObject_VAR_HEAD
```

Comment:
This makes any fixed sized executors larger. Although, I doubt it would be an issue in practice.

Reply:
Yeah, I had thought we could put the size field in the `_PyUOpExecutorObject` type only. But as you say, it doesn't matter in practice; and it means we can use the standard `PyObject_NewVar` pattern to allocate instances.